bedrock/firefox/templates/firefox/base.html

```diff
@@ -15,7 +15,7 @@
 {% block facebook_id %}14696440021{# facebook.com/Firefox #}{% endblock %}
 {% block twitter_id %}firefox{% endblock %}
 
-{% block site_css %}
+{% block page_css %}
```

Comment:
Hm, in `firefox/base-resp.html`, you leave this block as `site_css`. Any reason this template should be different?